doc/CloudIntegration.xml

```diff
@@ -0,0 +1,338 @@
+    <para>Figure <xref xmlns:xlink="http://www.w3.org/1999/xlink" linkend="_Ref752965479"/> demonstrates the two phases. N.B. the gray rectangle visually identifies the scope of these specifications.</para>
+    <figure xml:id="_Ref752965479">
+      <title>Onboarding phases and ONVIF scope</title>
+      <mediaobject>
+        <imageobject>
+          <imagedata fileref="media/CloudIntegration/onboarding_scope.svg" contentwidth="159.97mm" />
+        </imageobject>
+      </mediaobject>
+    </figure>
+    <section>
+      <title>Web Services </title>
+      <para>Unlike for other services defined by ONVIF, in this document the term Web Services is
+        the name of a standardized method of integrating applications using open, platform
+        independent Web Services standards such as JSON, OpenAPI 3.1.0 and YAML over an IP network.
+        JOSN is used as the data description syntax, OpenAPI is used for message transfer and YAML
+        is used for describing the services.</para>
+      <figure xml:id="_Ref211940201">
+        <title> OpenAPI based development principles</title>
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="media/CloudIntegration/openapi_based_development_principles.svg"
+              contentwidth="80mm"/>
+          </imageobject>
+        </mediaobject>
+      </figure>
+      <para>
+        <xref linkend="_Ref211940201"/> gives an overview of the basic principles for development
+        based on Web Services and OpenAPI. The service provider (MCS) implements the ONVIF service.
+        The service is described using the YAML-based file. Then, the YAML file is used as the basis
+        for the service requester (OCS) implementation/integration. Integration is simplified
+        through the use of code generating tools that generate platform specific code that can be
+        used by the developers to integrate the Web Service.</para>
+      <para>The Web Service provider and requester communicate using the OpenAPI message exchange
+        protocol. OpenAPI is a lightweight messaging protocol used to encode the information in a
+        Web Service request and in a response message before sending them over a network. OpenAPI
+        messages are independent of any operating system or protocol and may be transported using a
+        variety of Internet protocols. This ONVIF standard defines conformant transport protocols
+        for the SOAP messages for the described Web Services.</para>
+      <para>The Web Service overview section introduces into the general ONVIF service structure,
+        the command definition syntax in the specification, error handling principles and the
+        adopted Web Service security mechanisms.</para>
+      </section>
+      <section xml:id="_Ref208379139">
+        <title>Security</title>
+        <section>
+          <title>Authentication</title>
```

Comment:
Instead of this section we should probably have some requirement to define how the two cloud systems interact.
I'm thinking mandating HTTPS, and mandating that certificates are validating using publicly trusted authorities (and leaving that vague, to let different cloud providers have different set of authorities based on governmental requirements & such)

Reply:
I was thinking about using the same credentials that you use to access the MCS to claim the cameras. But I am open to discuss this topic.

Reply:
But, there's no credential when talking to the MCS. Otherwise we need to define how we get them. Right now, the credentials was the transfer token itself. Possession of the transfer token allowed a cloud system to claim the device, no further authentication was needed.
At least, that was my understanding of the proposal so far. Otherwise we need to figure out how two unknown cloud systems get to exchange credentials